config/application.yml.example

```diff
@@ -33,6 +33,7 @@ development:
   session_timeout_in: '15'
   telephony_disabled: 'true'
   twilio_accounts: '[{"sid":"sid", "auth_token":"token", "number":"9999999999"}]'
+  valid_service_providers: '["https://rp1.serviceprovider.com/auth/saml/metadata", "urn:gov:gsa:SAML:2.0.profiles:sp:sso:localhost", "urn:gov:gsa:SAML:2.0.profiles:sp:sso:localhost-rails", "https://dashboard.login.gov"]'
```

Comment:
Unfortunately, it cannot, which is why some of our other array-like keys use the JSON format. This is because Figaro only supports String values, just like regular ENV vars.